internal/runner/runner.go

```diff
@@ -238,3 +252,23 @@ func filterIP(options *Options, data Output) bool {
 	}
 	return false
 }
+
+func resolveToIP(domain string) (string, error) {
+	url, err := urlutils.Parse(domain)
+	if err != nil {
+		return domain, err
+	}
+	Options := fastdialer.DefaultOptions
+	dailer, err := fastdialer.NewDialer(Options)
+	if err != nil {
+		return domain, errorutils.New("%v: fialed to resolve domain", err.Error())
+	}
```

Comment:
Instead of creating fastdialer for every call, can we put into initialise as global variable or put inside runner. 
If require refactor the runner code .